netdata/netdata
# logs-management: Disable logs management monitoring section

##### Summary
This PR disables the Netdata monitoring section for logs management (unless Netdata monitoring has been enabled in `netdata.conf`).

Also, it removes files that I have been using for stress testing and development in the past and renames `LOGS_MANAGEMENT_STRESS_TEST` macro to `LOGS_MANAGEMENT_DEV_MODE`. 

##### Test Plan

- Install this PR an observe if after enabling logs management, it shows up in the `Netdata monitoring` section by default.
- Enable `netdata monitoring` and `netdata monitoring extended` configuration options and observe if there is a new `logsmanagement` monitoring section, when `logs-management.plugin` is enabled.

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
